packages/storybook/stories/INTERNAL/ClosedCaptions/ClosedCaptions.stories.tsx

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { CallComposite } from '@azure/communication-react';
+import { Stack } from '@fluentui/react';
+import { Description, Heading, Source, Subheading, Title } from '@storybook/addon-docs';
+import { Meta } from '@storybook/react/types-6-0';
+import React, { useMemo } from 'react';
+import { v1 as createGUID } from 'uuid';
+import { SingleLineBetaBanner } from '../../BetaBanners/SingleLineBetaBanner';
+import { ContosoCallContainer } from '../../CallComposite/snippets/Container.snippet';
+import { ConfigHintBanner } from '../../CallWithChatComposite/Utils';
+import {
+  COMPONENT_FOLDER_PREFIX,
+  compositeExperienceContainerStyle,
+  overviewPageImagesStackStyle
+} from '../../constants';
+import { ArgsFrom, controlsToAdd, defaultCallCompositeHiddenControls } from '../../controlsUtils';
+import { compositeLocale } from '../../localizationUtils';
+import { exampleDisableCaptions } from './ClosedCaptions';
+
+const getDocs: () => JSX.Element = () => {
+  /* eslint-disable react/no-unescaped-entities */
+  return (
+    <>
+      <Title>Captions</Title>
+      <Heading>Closed Captions</Heading>
+      <Description>
+        Azure Communication Services UI Library is adding support for Closed Captions. Closed captions is a powerful
+        tool that enables developers to enhance the accessibility of their videos. With closed captions, developers can
+        provide a textual representation of the audio content in their videos, making it easier for users who are deaf
+        or hard of hearing to follow along.
+      </Description>
+      <Description>
+        Closed Captions is enabled by default and are automatically included within the CallComposite and
+        CallWithChatComposite experiences.
+      </Description>
+
+      <Subheading>Interop Closed Captions</Subheading>
+      <Description>
+        Interop Closed Captions is enabled by default and are automatically included within the CallComposite and
```

Comment:
Suggestion: start with ACS captions, that covers captions being enabled by default, and what spoken language is etc.
Then have this Teams meeting interop section and have it call it the difference only and what it means for developers - when is it Teams interop captions? And what happens to the UI (suddenly there's a captions language option?)

Reply:
An alternative option is to have
```
## Spoken language

Detail what spoken language is and that an end user can change it.

## Captions Language

Detail what spoken language is and that an end user can change it. And that it's only available in Teams interop calls

### Captions in a Teams Meeting Interop Call

Detail when this kicks in and the functionality changes
```